1-exercises/F-for-of-loop/exercise.js

```diff
@@ -4,13 +4,19 @@
 
 // TODO Use a for-of loop to output each of the tube stations below.
 let tubeStations = [
-    "Aldgate",
-    "Baker Street",
-    "Picadilly Circus",
-    "Oxford Street",
-    "Tottenham Court Road"
+  "Aldgate",
+  "Baker Street",
+  "Picadilly Circus",
+  "Oxford Street",
+  "Tottenham Court Road",
 ];
 
+for (let stations of tubeStations) {
```

Comment:
Here you've used a plural variable name, it doesn't read well.  I would consider changing this just for readability purposes.  I think it would make more sense if it was named to station, tubeStop or even stationName.

Reply:
Corrected